ynput/OpenPype
# Houdini: Add loader for redshift proxy family

## Changelog Description
Loader for Redshift Proxy in Houdini (Thanks for @BigRoy contribution)

## Additional info
n/a

## Testing notes:
1. Launch Houdini
2. Create Redshift Proxy Instance
3. Publish
4. Load the redshiftproxy instance through loader
5. It should be loaded as expected.
